### PR TITLE
Add TOC and bookmarks to data collection

### PR DESCRIPTION
**Pull request by nCastle1**
Notes:

* Major UWP UI changes (driven by changes needed by WinUI to get treeview)
* Adds toolkit preview package on WPF

### CODE REVIEW

File: src/DataCollection.WPF/MainWindow.xaml [hunk 1]
```diff
@@ -222,26 +244,49 @@
...
-                    FontSize="25"
+                    FontSize="20"
                     Style="{StaticResource ToggleButtonStyle}"
                     ToolTip="{Binding ., Converter={StaticResource LocalizationConverter}, ConverterParameter=MoreTools_Tooltip}"
                     Visibility="{Binding IsLocationOnlyMode, Converter={StaticResource BoolToVisibilityConverter}, ConverterParameter=Inverse}" />
             </StackPanel>
         </Grid>
 
         <!--  Menu Panel  -->
+        <Grid Grid.Row="1"
```
Comment (ZackAllen) [inline src/DataCollection.WPF/MainWindow.xaml]: I feel like the TOC would really benefit from some type of border. It feels unfinished with the white background with no rounded corners.

File: src/DataCollection.WPF/MainWindow.xaml [hunk 2]
```diff
@@ -222,26 +244,49 @@
...
+              Grid.RowSpan="2"
+              Grid.Column="2"
+              MaxWidth="250"
+              MaxHeight="250"
+              Margin="0,5,5,0"
+              Background="White"
+              VerticalAlignment="Top"
+              Visibility="{Binding IsChecked, ElementName=toggleTOC, Converter={StaticResource BoolToVisibilityConverter}}">
+            <esri:TableOfContents GeoView="{Binding ElementName=MapView, Mode=OneWay}" ShowLegend="False" />
```
Comment (ZackAllen) [inline src/DataCollection.WPF/MainWindow.xaml]: If possible, changing the font family here could look better.

File: src/DataCollection.WPF/MainWindow.xaml [hunk 3]
```diff
@@ -222,26 +244,49 @@
...
             </StackPanel>
         </Grid>
 
         <!--  Menu Panel  -->
+        <Grid Grid.Row="1"
+              Grid.RowSpan="2"
+              Grid.Column="2"
+              MaxWidth="250"
+              MaxHeight="250"
+              Margin="0,5,5,0"
```
Comment (ZackAllen) [inline src/DataCollection.WPF/MainWindow.xaml]: I think this needs a larger vertical margin.

File: src/DataCollection.UWP/MainPage.xaml [hunk 4]
```diff
@@ -101,19 +42,72 @@
...
             Foreground="{x:Bind MainViewModel.ConnectivityMode, Mode=OneWay, Converter={StaticResource ConnectivityModeToColorConverter}, ConverterParameter=Inverse}">
             <CommandBar.Content>
                 <TextBlock
                     Margin="15,0,0,0"
-                    Style="{StaticResource SubheaderTextBlockStyle}"
+                    Style="{StaticResource SubheaderTextBlockStyle}" FontSize="28"
                     Text="{x:Bind MainViewModel.MapViewModel.Map.Item.Title, Mode=OneWay}" />
             </CommandBar.Content>
+            <AppBarButton
+                x:Uid="ShowBookmarksButton"
```
Comment (ZackAllen) [inline src/DataCollection.UWP/MainPage.xaml:67]: After downloading a map, having a bookmark to return to the view of the offline section would be useful. When downloading a small section, then navigating away with a bookmark, it is hard to find the offline section where a map is visible.

**Review by ZackAllen — CHANGES_REQUESTED** [review]
The add button on UWP does not work on my machine.

**Comment by ZackAllen** [comment]
DataCollection WPF_cZ0N2Qw3SI

ToC does not align to the side when using the WPF app in full screen.

**Review by ZackAllen — APPROVED** [review]
Drop shadow helped WPF a lot.

**Comment by esreli** [comment]
A tip, take it or leave it. You might consider adding lines to the README as you go, that way the RELEASE notes aggregate as you add to the project rather than having to go back at the end and remember what work was completed.

**Comment by nCastle1** [comment]
@ZackAllen Can you please re-review? I made some UI changes and I'd like to verify they're good before retaking screenshots.

**Review by ZackAllen — APPROVED** [review]
Both UIs look great

**Pull request merged**